### PR TITLE
Removed delegation

### PR DESCRIPTION
**Pull request by David-Chadwick**
Rejigged the Subject-Holder Relationship section to remove the contentious delegation section and put it in a non-normative Appendix as an example.

***
<a href="https://pr-preview.s3.amazonaws.com/David-Chadwick/vc-data-model/pull/281.html" title="Last updated on Nov 15, 2018, 3:41 PM GMT (5cb8411)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-data-model/281/99b0dc9...David-Chadwick:5cb8411.html" title="Last updated on Nov 15, 2018, 3:41 PM GMT (5cb8411)">Diff</a>

### CODE REVIEW

**Comment by dlongley** [comment]
@David-Chadwick, from looking at the diff, it seems like this PR is doing more than just removing delegation? Can you make sure you're basing it off of the most recent version of the spec? You should be able to use git tools to make sure your fork is up to date, but if they're giving you too much trouble you could "hard reboot" by reforking the repo.

**Comment by David-Chadwick** [comment]
It is based on the very latest version that I took earlier this afternoon. If you look carefully, you will see that I have also simply moved (but not edited) the "Subject only terms of use" section from the "Subject Passes the Verifiable Credential to a Holder" section the  to the "Subject is Holder" section (where it more correctly belongs), because the section it was in has effectively been removed to the appendix.

File: index.html [hunk 1]
```diff
@@ -220,16 +220,6 @@ <h3>What is a Verifiable Credential?</h3>
...
-        <p>
-            The persistence of digital information, and the ease with which
-            disparate sources of digital data may be collected and correlated,
-            comprise a privacy concern that the use of verifiable and easily
-            machine-readable credentials threatens to exacerbate. A <a href="#privacy-considerations">Privacy Considerations</a>
-            section addresses these issues. Items of particular concern in
-            this specification may also be noted in the text. Examples of
-            how to use this data model using privacy-enhancing technologies
-            such as zero-knowledge proofs are also provided.
-        </p>
```
Comment (dlongley) [inline index.html]: @David-Chadwick, did you mean to remove this?

File: index.html [hunk 2]
```diff
@@ -1965,7 +1880,7 @@ <h2>Holder Acts on Behalf of the Subject</h2>
       <pre class="example nohighlight" title="An example of the relationship
 property in a child's credential">{
   "id": "http://dmv.example.gov/credentials/3732",
-  "type": ["VerifiableCredential", "ProofOfAgeCredential"],
+  "type": ["VerifiableCredential", "ChildProofOfAgeCredential"],
```
Comment (dlongley) [inline index.html]: Did you mean to change this?

File: index.html [hunk 3]
```diff
@@ -2102,55 +2017,26 @@ <h2>Disputes</h2>
...
-the disputed credential.  For example, if a credential with an identifier of
-<code>https://example.org/credentials/245</code>
-is disputed, an entity may issue one of the following credentials. In the
-former case, the subject might present this to the verifier along with the
-disputed credential. In the latter case, the entity might publish the
-"DisputeCredential" in a public venue to make it known that the credential
-is disputed.
-        </p>
-
-        <pre class="example nohighlight" title="Subject disputes a credential">
```
Comment (dlongley) [inline index.html]: Did you mean to remove all this? It seems like a reversion of the current spec and the PR that was pulled in to clean this DisputeCredential stuff up.

File: index.html [hunk 4]
```diff
@@ -3397,7 +3270,84 @@ <h3>Device Theft and Impersonation</h3>
         </ul>
 
     </section>
+      <section>
+        <h2>Accessibility Impact</h2>
+        <p>There are a number of accessibility considerations of which implementors should be aware when processing data described in this specification. As with any implementation of web standards or protocols, ignoring accessibility issues will make this information unusable to a large subset of the population. It is important to follow accessiblity guidelines and standards such as [WCAG21] to ensure that all people, regardless of ability can make use of this data. This is especially important when establishing cryptography and encryption which historically have created problems for assistive technologies.
```
Comment (dlongley) [inline index.html]: If this was intentionally moved, can you restore the formatting to what was removed above?

**Comment by dlongley** [comment]
@David-Chadwick, I left a few comments on the sections that didn't seem like you intentionally removed, i.e., the ones that appear to reversions to an older version of the spec.

File: index.html (hunk 4 above)
Comment (dlongley) [reply index.html]: Actually, this also appears to be a reversion of another commit that moved the accessibility section: https://github.com/w3c/vc-data-model/commit/99b0dc99ce99878e63de6a188ea11edb6e6cdad3

File: index.html (hunk 1 above)
Comment (David-Chadwick) [reply index.html]: Certainly not. If I did I truly apologise

File: index.html (hunk 2 above)
Comment (David-Chadwick) [reply index.html]: Yes, I made this comment many weeks ago in an issue and it seemed to get overlooked.
Rationale: the contents of this VC are different from the other VCs that are proof of age, therefore the type should be different

File: index.html (hunk 3 above)
Comment (David-Chadwick) [reply index.html]: No I did not mean to remove this. This was a mistake

File: index.html (hunk 4 above)
Comment (David-Chadwick) [reply index.html]: I will try to fix my mistakes now. Sorry for the finger trouble

File: index.html (hunk 1 above)
Comment (David-Chadwick) [reply index.html]: I have just checked the master copy that I pulled as the above text is not in my original version. So I did not edit it out.

File: index.html [hunk 5]
```diff
@@ -1851,76 +1770,72 @@ <h3> 'Subject Only' Terms of Use </h3>
...
-the issuer is the subject,
-the subject is the holder to whom the verifiable credential is being passed,
-and the claim contains the properties that are being passed on. In addition, the
-holder creates a verifiable presentation that contains these two
-verifiable credentials.
+      <p>
+In this case, the <a>claim</a> may contain multiple properties that each
+provide an aspect of the identity of the subject, and which together
+unambiguously identify the subject. 
 </p>
```
Comment (David-Chadwick) [inline index.html]: The above change was not made by myself, but was already in the master copy that I pulled

File: index.html (hunk 3 above)
Comment (David-Chadwick) [reply index.html]: I have just checked my master copy and I did not remove this. It was already removed.
So I guess I did not edit the latest version.
It is probably better to simply remove my PR and I will start all over again.

File: index.html (hunk 4 above)
Comment (David-Chadwick) [reply index.html]: Can we simply delete this PR and I will start all over again with a new master copy

**Comment by David-Chadwick** [comment]
Please delete this PR as it has been replaced by #283 
Thankyou

**Pull request closed without merging**